pkg/objectfile/object_file_test.go

```diff
@@ -82,14 +80,6 @@ func TestComputeBase(t *testing.T) {
 			addr:      0x1000,
 			wantError: true,
 		},
-		{
```

Comment:
Fair, I removed it because we are no longer calling `elf.Open()` from `computeBase`; shall I create a new `TestOpenElf` and add it there (and move `TestOpenMalformedElf` in there too?)

Reply:
That would be perfect!